blockstack-sdk/src/main/java/org/blockstack/android/sdk/BlockstackSession.kt

```diff
@@ -629,12 +646,21 @@ class BlockstackSession(context: Context? = null, private val config: Blockstack
     }
 }
 
+/**
+ * Executor defines where functions are executed. Three different cases are distinguished:
+ * 1. main thread: to start an intent launching the login process. This has to be the UI thread
+ * 1. network thread: to make network calls. This must not be the UI thread, it is usually a thread from CommonPool
```

Comment:
Minor nit: Add period at the end of these comments, just to be consistent with full sentences :).